openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -77,6 +78,38 @@ class Config:
     endstate the torsion term is turned off/on depending on the state the
     unique atoms belong to. Default False.
     """
+    explicit_charge_correction = True
+    """
+    Whether to explicitly account for a charge difference during the
+    alchemical transformation by transforming a water to a counterion
+    of the opposite charge of the formal charge difference.
+
+    Please note that absolute charge changes greater than 1 are
+    currently not supported.
+    """
+    explicit_charge_correction_cutoff = 0.8 * unit.nanometer
+    """
+    The minimum distance from the system solutes from which an
+    alchemical water can be chosen. Default 0.8 * unit.nanometer.
+    """
+    alchemical_water_ion_positive_resname = 'NA'
+    """
+    Name of the positive ion used to replace the alchemical water with
+    if ``explicit_charge_correction`` is ``True`` and the absolute charge
+    difference is negative.
+    """
+    alchemical_water_ion_negative_resname = 'CL'
+    """
+    Name of the negative ion used to replace the alchemical water if
+    ``explicit_charge_correction`` is ``True`` and the absolute charge
+    difference is positive.
+    """
+    alchemical_water_resname = 'HOH'
```

Comment:
this is probably not a real setting, isn't this just the resname that `add_solvent()` adds?  (i.e. it's know a priori and can be hard coded?)

Reply:
Todo - I need to double check the FF files to see if the resname is inherited or set by modeller.


However! This does bring up an interesting thought, this won't work with 4+ site waters because we don't handle vsite transformations. I need to add in a guard & some docs for it.

Reply:
done on both counts